package/scripts/auth.py

```diff
@@ -13,7 +13,6 @@
 # the License.
 #
 
-import sys
 import ambari_helpers as helpers
 from resource_management import *
```

Comment:
I have always understood this (``import *``) to be a poor practice, because then you can have namespace collisions that are really difficult to debug. Better to either import something specifically (``import ambari_helpers as helpers``), or import the entire module (``import sys``) and refer to its objects with the complete name (``sys.path``).